### DISCUSSION TITLE
Weird issue with rust-analyzer and google-apis-rs

### DISCUSSION THREAD

**Discussion by dlen** [Byron/google-apis-rs#483] [General]
Hey folks,

I have opened this discussion in the rust-analyzer forum with little luck https://github.com/rust-lang/rust-analyzer/discussions/16999 I'm wondering if you could reproduce this issue and if it has something to do with conflicting versions somehow in the crates I list in the above link or is just my computer(tm).

**Comment by Byron** [comment]
[Since they seem to be picking it up](https://github.com/rust-lang/rust-analyzer/discussions/16999#discussioncomment-8994837), I think there is no need to deal with rust-analyzer here.

**Comment by cc8dea2e-4a29-4394-af6d-af0ccc73647a** [comment]
@dlen Hi! Did you manage to find a solution to this problem? I have this behavior ra too, when using google-bigquery2 crate

**Comment by dlen** [comment]
Hey, no not really. I basically stopped using these crates :(
